adobe_vipm/flows/global_customer.py

```diff
@@ -467,7 +475,9 @@ def create_gc_agreement_subscription(
             ]
         },
         "externalIds": {"vendor": adobe_subscription["subscriptionId"]},
-        "lines": [{"quantity": adobe_subscription["currentQuantity"], "item": item}],
+        "lines": [{"quantity": adobe_subscription["currentQuantity"], "item": item,
```

Comment:
What type is `prices` here?. It looks like it is just a price of the item and that it.
In that case why do you need so complex logic on merging dicts???
Why not to:

```
"lines": [
  {
    "quantity": adobe_subscription["currentQuantity"],
    "item": item,
    "price": {"unitPP": price}
  }]
``` 
???